languages/python/oso/polar/host.py

```diff
@@ -17,6 +17,62 @@
 from .expression import Expression, Pattern
 
 
+# Entity classes
+
+
+class OsoResource:
+    pass
+
+
+class OsoActor:
+    pass
+
+
+class OsoGroup:
+    pass
+
+
+class EntityMap:
+    ENTITY_TYPES = {
```

Comment:
I don't quite see why these need to be stringified, and then the API takes in a string. Why not have the user import these as classes or constants?